content/accessibility/headings.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: Headings
+---
+
+## Overview
+
+Headings play a critical role in communicating the structure of a page. Heading levels range from `<h1>` to `<h6>`. 
+
+### Best practices
+
+- Avoid skipping heading levels. `<h1>` should be followed by `<h2>` and so on.
+- Do not use heading elements solely for resizing text.
+- Avoid setting more than one `<h1>` per page. `<h1>` should be reserved to describe the page as a whole, similar to a page `<title>`.
```

Comment:
This might be too niche to include here, but I wonder if we should add a note about dialogs having and `h1` like this [WIP example](https://github.com/github/github/pull/208416/files#diff-e681003f10b8c77bc4f58acf418117637df345a2879bc2a457e5ccc9cbee5aebR8). Initially I thought this would break the one `h1` per page, but since its kind of like its own "page" its good practice (at least that's my understanding). I could see other engineers questioning that dialog `h1` in the future, though.

Reply:
Good callout! Let's add that note.